Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -2114,13 +2114,21 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
   fillStrangenessTrackingTables(recoData, trackedV0Cursor, trackedCascadeCursor, tracked3BodyCurs);
 
   // helper map for fast search of a corresponding class mask for a bc
+  pc.inputs().get<o2::ctp::CTPConfiguration*>("ctpconfig");
```

Comment:
Better move this here: https://github.com/AliceO2Group/AliceO2/blob/5f1b3b625bb2097cb20a5193b1daa157a532e010/Detectors/AOD/src/AODProducerWorkflowSpec.cxx#L2547

Reply:
Done